### PR TITLE
Cog 1436 office admin user acceptance test

### PR DESCRIPTION
**Pull request by kelfink**
#### Which User story does this relate to (link)?
- COG-1436 acceptance test
#### Brief feature description
- Some refactoring of helper method names
- Splitting of tests into office and county for different users
- Improvements to detecting when verification code entry is completed

### CODE REVIEW

File: spec/acceptance/county/user_login_spec.rb [hunk 1]
```diff
@@ -92,8 +92,7 @@
     expect(page)
       .to have_text('Error. Incorrect code. You have 2 attempts remaining.')
     verify_account
-    go_manage_users
```
Comment (RamuKammagani-osi) [inline spec/acceptance/county/user_login_spec.rb]: We still need go_manage_users to land on Search Page from dashboard! isn't it??

Comment (kelfink) [reply spec/acceptance/county/user_login_spec.rb]: ah, maybe!  This might explain the PR failures.  Let me try....

**Comment by kelfink** [comment]
Made changes to the API requests, so it grabs them relative to the /cap which is not always in place in the various environments.

File: spec/acceptance/county/user_login_spec.rb (hunk 1 above)
Comment (kelfink) [reply spec/acceptance/county/user_login_spec.rb]: Failure in the PR ended up being URL confusion in the tests.  was directing to the wrong place.  See *green* builds now.

Comment (kelfink) [reply spec/acceptance/county/user_login_spec.rb]: PR issue was fixed by URL changes in later commits.

**Pull request merged**